apps/server/src/utils/fileManagement.ts

```diff
@@ -114,3 +109,20 @@ export async function dockerSafeRename(oldPath: PathLike, newPath: PathLike) {
   await copyFile(oldPath, newPath);
   await unlink(oldPath);
 }
+
+/**
+ * finds potential file index number in our (*) format and increments
+ */
+export function incrementProjectNumber(path: string): string {
+  const { dir, name, ext } = parse(path);
+
+  if (!name.endsWith(')')) return join(dir, `${name} (1)${ext}`);
```

Comment:
question: is there a reason why we cant just demand the existence of extension?

Reply:
not sure i understand here?
Am I not just assuming the extension?

Reply:
yes, you are correct.

if the path is not really a path and there is no extension, the logic will break